explainer-shared/src/main/scala/shared/models/ClientSideModels.scala

```diff
@@ -25,7 +25,8 @@ object CsAtom extends ExplainerAtomImplicits {
   private implicit def changeRecordToCsChangeRecord(changeRecord: Option[ChangeRecord]): Option[CsChangeRecord]= changeRecord.map(cr => CsChangeRecord(cr.date, cr.user))
   private implicit def contentChangeDetailsToCsContentChangeDetails(contentChangeDetails: ContentChangeDetails): CsContentChangeDetails =
     CsContentChangeDetails(contentChangeDetails.lastModified, contentChangeDetails.created, contentChangeDetails.published, contentChangeDetails.revision)
-  private implicit def explainerAtomToCsExplainerAtom(explainerAtom: ExplainerAtom): CsExplainerAtom = CsExplainerAtom(explainerAtom.title, explainerAtom.body, explainerAtom.displayType.name)
+  private implicit def optionSeqStringToOptionListString(tags: Option[Seq[String]]): Option[List[String]] = tags.map( x => x.toList )
+  private implicit def explainerAtomToCsExplainerAtom(explainerAtom: ExplainerAtom): CsExplainerAtom = CsExplainerAtom(explainerAtom.title, explainerAtom.body, explainerAtom.displayType.name, explainerAtom.tags)
```

Comment:
In this case as the transformation is so simple I'd suggest you just write it out rather than using an implicit, a bit like displayType. So:
`......explainerAtom.displayType.name, explainerAtom.map(_.toList))`